client/src/pages/Home.tsx

```diff
@@ -0,0 +1,71 @@
+import NavigationBar from "../components/NavigationBar.tsx";
+import { Button, Col, Container, Row, Stack } from "react-bootstrap";
+import SolutionCard, {
+  SolutionCardProps,
+} from "../components/SolutionCard.tsx";
+import { ArrowUpRightSquare } from "react-bootstrap-icons";
+import ChallengeCard, {
+  ChallengeCardProps,
+} from "../components/ChallengeCard.tsx";
+import Footer from "../components/Footer.tsx";
+
+const DEMO_SOLUTION_CARDS: SolutionCardProps[] = Array(5).fill({
+  title: "Example Challenge 1: Airport Management System",
+  description:
+    "Some quick example text to build on the card title and make up the bulk of the card's content.",
+  imgSrc:
+    "https://images.unsplash.com/photo-1596496181871-9681eacf9764?q=80&w=2086&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
+  href: "/home",
+});
+
+const DEMO_CHALLENGE_CARDS: ChallengeCardProps[] = Array(3).fill({
+  title: "Demo Challenge Card",
+  description:
+    "In this demo challenge you will be demoing our platform. From creating UML diagrams to submitting" +
+    " and getting review, we provide you all you need to become a Software Architecture monster!",
+  href: "/home",
+});
+
+function Home() {
+  return (
+    <section>
+      <NavigationBar />
+      {/* Recent Solutions */}
+      <Container className={"mt-5"}>
+        <h2 className={"mb-3"}>Recent Solutions</h2>
+        <Row sm={1} lg={3} className={"gx-4 gy-4"}>
+          {DEMO_SOLUTION_CARDS.map((props) => (
```

Comment:
Could this be made into an element? So you would just pass the props, without calling map. I am not sure if that is possible, but it would be nice.